build/scripts/download-nuget.ps1

```diff
@@ -27,6 +27,7 @@ try {
 
     Write-Host "Downloading NuGet.exe"
     $webClient = New-Object -TypeName "System.Net.WebClient"
+    $webClient.Proxy.Credentials = [System.Net.CredentialCache]::DefaultNetworkCredentials
```

Comment:
❓ Should we be using this pattern instead?

http://stackoverflow.com/a/20369351/138304